projects/packages/sync/src/modules/class-module.php

```diff
@@ -317,17 +317,17 @@ public function get_next_chunk( $config, $status, $chunk_size ) {
 	 */
 	public function get_last_item( $config ) {
 		global $wpdb;
-		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery,Generic.WhiteSpace.ScopeIndent.Incorrect
```

Comment:
Do we still need this change now that the indent is fixed?

Reply:
Good catch. I removed one exception but missed a second one. Updated in [8a86004](https://github.com/Automattic/jetpack/pull/37396/commits/8a86004da67647f07580f5962185e1c354d4a949)